docs/meshstack.billing-configuration.md

```diff
@@ -94,6 +94,26 @@ let example2 =
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+
+#### Usage Types and Units
+
+When configuring a product for private cloud billing, operators need to choose the cloud resource type to target and how the metering engine
+should generate usages and apply prices to them. The documentation section of each private cloud platform lists the supported resource types and the traits available in the metering engine.
+Traits are a properties of a cloud resource like CPU or RAM. Operators can use traits to configure predicates (filter resources applicable to a pricing rule) and to control how meshMetering calculates usages.
+
+meshMetering supports flexible rules for creating usages, like
+
+- **time**: charge consumption based on the usage of a resource over a time period
+- **quantity**: charge consumption based on a quantity
+- **time-quantity**: charge consumption based on the product of time and quantity
+
+Quantities are represented with prefixable units according to UCUM standard inside meshMetering.
+For example, one kilo byte would be represented as `1 kB` or one kibi-byte as `1 kiBy`.
```

Comment:
q: Why is it not `1 kiB` even though it's byte in both cases?

Reply:
We are using the UCUM standard https://ucum.org/ucum.html and §48 lists that for bytes its "By". (theres instead a mistake there in 1 kB). 

I will make this section more precise
- we use "print" units for formatting where units need to be human readable (e.g. in usage reports)
- we use the case-sensitive unit representations when units are processed automatically (traits, product catalog) and need to be serializable